aws_google_auth/__init__.py

```diff
@@ -371,7 +371,7 @@ def parse_args(args):
     parser.add_argument('-d', '--duration', type=int, default=DURATION,
                         help='Credential duration ($DURATION)')
     parser.add_argument('-p', '--profile', default=PROFILE,
-                        help='AWS profile ($AWS_PROFILE)')
+                        help='AWS profile ($AWS_PROFILE) (defaults to \'sts\')')
```

Comment:
This will default to the env var `AWS_PROFILE`, if that does not exist then could fall back to `sts`.